demos/object_detection_demo/python/object_detection_demo.py

```diff
@@ -139,8 +151,9 @@ def __len__(self):
 
 
 def get_model(ie, args):
+    common_args = (ie, args.model, args.reverse_input_channels, args.mean_values, args.scale_values)
     if args.architecture_type == 'ssd':
-        return models.SSD(ie, args.model, labels=args.labels, keep_aspect_ratio_resize=args.keep_aspect_ratio)
+        return models.SSD(*common_args, labels=args.labels, keep_aspect_ratio_resize=args.keep_aspect_ratio)
     elif args.architecture_type == 'ctpn':
         return models.CTPN(ie, args.model, input_size=args.input_size, threshold=args.prob_threshold)
```

Comment:
If a model doesn't support input transforms, then the demo should produce an error if the user tries to use it with transforms.

Reply:
Done.